src/m365/spo/commands/site/site-set.spec.ts

```diff
@@ -810,6 +845,42 @@ describe(commands.SITE_SET, () => {
     });
   });
 
+  it('updates description of the specified groupified site', (done) => {
+    sinon.stub(request, 'get').callsFake((opts) => {
+      if (opts.url === 'https://contoso.sharepoint.com/sites/Sales/_api/site?$select=GroupId,Id') {
+        return Promise.resolve({
+          Id: '255a50b2-527f-4413-8485-57f4c17a24d1',
+          GroupId: 'e10a459e-60c8-4000-8240-a68d6a12d39e'
+        });
+      }
+
+      return Promise.reject('Invalid request');
+    });
+    sinon.stub(command as any, 'getSpoAdminUrl').callsFake(() => Promise.resolve('https://contoso-admin.sharepoint.com'));
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if (opts.url === 'https://contoso-admin.sharepoint.com/_api/SPOGroup/UpdateGroupPropertiesBySiteId' &&
+        JSON.stringify(opts.data) === JSON.stringify({
+          groupId: 'e10a459e-60c8-4000-8240-a68d6a12d39e',
+          siteId: '255a50b2-527f-4413-8485-57f4c17a24d1',
+          displayName: 'New title'
+        })) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.action(logger, { options: { debug: false, title: 'New title', description: 'Some description', url: 'https://contoso.sharepoint.com/sites/Sales' } }, () => {
+      try {
+        assert(loggerLogToStderrSpy.called);
```

Comment:
ok didn't notice this but actually it not resolves and the done is not reached 
![image](https://user-images.githubusercontent.com/58668583/114665728-a8f1a600-9cfd-11eb-83b2-1a60d31c26cf.png)
breaks gh actions with timeout. To recheck 👨‍💻

Reply:
I marked the PR as draft. When you're ready and we have a working PR build, please change it to 'Ready for review'. Appreciate your help! 👏